extensions/2.0/Vendor/EXT_feature_metadata/examples/microcosm/microcosm.gltf

```diff
@@ -0,0 +1,289 @@
+    "EXT_feature_metadata"
+  ],
+  "extensions": {
+    "EXT_feature_metadata": {
+      "extras": {
+        "draftVersion": "0.0.0"
+      },
+      "featureClasses": {
+        "NDVI": {
+          "name": "Normalized difference vegetation index",
+          "semantic": "NDVI",
+          "properties": {
+            "NDVI": {
+              "name": "Normalized Difference Vegetation Index",
+              "semantic": "NDVI",
+              "elementType": "UINT8",
+              "normalized": true
+            }
+          }
+        },
+        "LULC": {
+          "name": "Landcover/Landuse",
+          "semantic": "LULC",
+          "properties": {
+            "Name": {
+              "name": "Name",
+              "semantic": "NAME",
+              "elementType": "STRING"
+            },
+            "Color": {
+              "name": "Color",
+              "semantic": "COLOR",
+              "elementType": "ARRAY",
+              "componentType": "UINT8",
+              "componentsPerElement": 3
+            }
+          }
+        }
+      },
+      "featureTables": {
+        "LandCover": {
+          "featureClass": "LULC",
+          "featureCount": 6,
+          "elementCount": 256,
+          "properties": {
+            "Name": {
+              "bufferView": 5,
+              "offsetBufferViews": [6]
+            },
+            "Color": {
+              "bufferView": 7
+            }
+          }
+        }
+      }
+    }
+  },
+  "scene": 0,
+  "scenes": [
+    {
+      "name": "Scene",
+      "nodes": [0]
+    }
+  ],
+  "nodes": [
+    {
+      "mesh": 0,
+      "name": "Terrain"
+    }
+  ],
+  "materials": [
+    {
+      "doubleSided": true,
+      "name": "Photogrammetry",
+      "pbrMetallicRoughness": {
+        "baseColorTexture": {
+          "index": 0,
+          "texCoord": 0
+        },
+        "metallicFactor": 0,
+        "roughnessFactor": 0.5
+      },
+      "extensions": {
+        "EXT_feature_metadata": {
+          "textures": [
+            {
+              "name": "NDVI",
+              "index": 1,
+              "texCoord": 1
+            },
+            {
+              "name": "Land Cover",
+              "index": 2,
+              "texCoord": 1
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "meshes": [
+    {
+      "name": "Terrain Mesh",
+      "primitives": [
+        {
+          "attributes": {
+            "POSITION": 0,
+            "NORMAL": 1,
+            "TEXCOORD_0": 2,
+            "TEXCOORD_1": 3
+          },
+          "indices": 4,
+          "material": 0
+        }
+      ]
+    }
+  ],
+  "textures": [
+    {
+      "name": "Photogrammetry Texture",
+      "source": 0
+    },
+    {
+      "name": "NDVI Texture",
+      "source": 1,
+      "extensions": {
+        "EXT_feature_metadata": {
+          "features": [
+            {
+              "featureClass": "NDVI",
+              "properties": {
+                "NDVI": {
+                  "channels": "r"
```

Comment:
Another thing, I find it kinda confusing that this channel mapping is listed here in the `textures` section of the glTF. Especially if we need something more complex like byte order, I'd rather list this up in the top level extension so we'd have the three properties:

```
featureClasses: {
  // conceptual classes and the properties that belong to them
}
featureTables: {
  // determines what bufferViews to use for storing basic properties.
  // the actual bufferViews are indexed by feature ID
}
featureTextures: { // or maybe textureTables?
    // determines what textures to use for storing texture properties
    "NDVI": {
        "properties": {
            "NDVI": [
                {
                    "texture": 0,
                    "channels": [0], 
                }
            ],
            "SomeVec3PropertySplitAcrossTwoTextures": [
                 {
                     "texture": 1,
                      "channels": [0, 1, 2] // more future proof to use numbers rather than rgba? or is it better to stick to the rgba convention for clarity since we're limited to PNG and JPEG?
                 },
            ],
            "SomeUint64Property": [
                 // uint64 requires 8 channels. channels are listed in some standard order
                 // little endian maybe? or would big endian be easier to understand?
                 {
                     "texture": 2,
                     "channels": [0, 1, 2, 3]
                 },
                 {
                     "texture": 3,
                     "channels": [0, 1, 2, 3]
                 }
            ],
            "UnusualDataOrganization": [
                // vec4 but split across 3 textures.
                 {
                     "texture": 4,
                     "channels": [0]
                 },
                 {
                     "texture": 5,
                     // GB
                     "channels": [2, 1]
                 }
                 {
                     "texture": 6,
                     "channels": [0]
                 }
              ]
        }
    }
}
```

Reply:
My preference is to keep texture metadata as simple as possible by enforcing a `channel = component` relationship

Reply:
And avoid splitting components across multiple textures until there's a strong use case to do otherwise